docs/src/pages/index.js

```diff
@@ -627,28 +627,53 @@ export default function IndexPage() {
           />
           <div className="container20">
             <div className="container21">
-              <h2 className="text-white md:text-left lg:text-44 text-3xl text-center font-bold">
+              <h2 className="text-white md:text-left lg:text-32 text-2xl mt-6 text-center font-bold">
                 Get Started with Monika Now!
               </h2>
-              <p className="text-white font-medium lg:text-2xl lg:w-3/4 w-full md:my-0 my-4 md:text-left text-center">
+              <p className="text-white font-medium lg:text-lg lg:w-11/12 w-full my-5 md:text-left text-center">
                 Monika is an open source synthetic monitoring command line
-                application. It is actively developed and completely free!
+                application. it is actively developed and completely free!
               </p>
+              <div className="buttonFooter">
+                <Link href="/quick-start">
+                  <a
+                    className="link25 button"
+                    onClick={() =>
+                      event({
+                        action: 'cta_button_footer',
+                        category: 'cta_button',
+                      })
+                    }
+                  >
+                    Start Now
+                  </a>
+                </Link>
+              </div>
             </div>
             <div className="container22">
-              <Link href="/quick-start">
-                <a
-                  className="link15 button"
-                  onClick={() =>
-                    event({
-                      action: 'cta_button_footer',
-                      category: 'cta_button',
-                    })
-                  }
-                >
-                  Start Now
-                </a>
-              </Link>
+              <h2 className="text-white md:text-left lg:text-32 text-2xl text-center font-bold">
```

Comment:
![image](https://github.com/hyperjumptech/monika/assets/16670475/1fba92cd-ae6a-4417-9012-a2b14f3bb0a8)

Another improvement: The text should be at the top, while the button should be at the bottom.

Example:
![image](https://github.com/hyperjumptech/monika/assets/16670475/28112295-34d9-4332-a739-a13bd3914fc7)

So no matter how long the text is, the button is always at the bottom.